src/libs/test/jest/matchers.cljs

```diff
@@ -0,0 +1,36 @@
+(ns libs.test.jest.matchers)
+
+(js/expect.extend
+ #js{:toEq
+     (fn toEq [received expected]
```

Comment:
[Small improvement] This could be a proper named function with a docstring.

Reply:
Yeah this would be nice to have. To be honest I took this code from here: https://github.com/pesterhazy/shadow-jest/blob/master/src/test/jest/matchers.cljs

I'm not sure whether we fully need it yet as it just adds some additional utility to jests expect function.
I would like to get this code merged in first and have devs writing tests and I can revisit this after wards.

I also need to add documentation on how to run/write these tests.

Reply:
Sure, sounds good! It's just that it's a lot harder to take things away once merged, so if this matcher is not really needed we could add it only when we're certain of its value.

Reply:
Yeah I think I prefer your approach. So I will remove it from this p.r.